pkg/operator/operator.go

```diff
@@ -33,6 +33,8 @@ func Cmd() *cobra.Command {
 		Use:   "operator",
 		Short: "Deployment using operator",
 	}
+	// health-probe flag available on operator and inherited by subcommands
+	cmd.PersistentFlags().String("health-probe-bind-address", ":8081", "HTTP address for health/readiness probes (e.g., :8081)")
```

Comment:
Is it necessary to have a flag for the probe address? If there isn't a need to change it for some cases, we can have it defined in manager.go (e.g.: similar to the metrics address\port [here](https://github.com/OdedViner/noobaa-operator/blob/6095110f8da24bfe9b7a291aba62c7d5927d4d0f/pkg/operator/manager.go#L35-L37))